db/seeds.rb

```diff
@@ -102,3 +121,24 @@
     })
   end
 end
+
+puts "Creating Test Parent..."
+
+test_parent_user = User.create(
+  first_name: "TestParent",
+  last_name: "TEST",
+  email: "parent@campzone.com",
+  password: "password",
+  role: "parent"
+)
+
+test_parent = Parent.create(
+  user_id: test_parent_user.id,
+  address: "123 Test Street",
+  phone_number: "5141234567"
+)
+
+KidsParent.create(
+  parent_id: test_parent.id,
+  kid_id: 3
```

Comment:
Are you assigning the 3rd kid to this parent? Does that mean that they will have two parents? Not a problem, just asking! :-)